01-Login/src/auth/authWrapper.js

```diff
@@ -73,9 +73,8 @@ export const useAuth0 = ({
     },
     async created() {
       this.auth0Client = await createAuth0Client({
-        domain: options.domain,
+        ...options,
```

Comment:
Just out of curiosity, why did we move to spreading the options? Even tho it is not problematic, we are padding an object that has both `clientId` and `client_id` as a property.